Core/Inc/RCUv2.h

```diff
@@ -29,25 +29,29 @@ class RCUv2: public GenericChannel
 		int init() override;
 		//int reset() override;
 		int exec() override;
-
 		void testIMU();
 		void testGNSS();
+		void beep(int frequency, int length, int delay);
+		void startupBeep();
+		void superMario();
+		void gnssBeep();
 
 	private:
 		STRHAL_GPIO_t led1, led2;
 
 		// Modules
 		LPS25HB_Baro baro;
 		ICM2060x_IMU imu;
-		LoRa1276F30_Radio lora;
+		SX1276 lora;
 		SAM_M8Q_GNSS gnss;
 
 		// Channels
 		ADCChannel sense_5V, sense_12V;
 		BaroChannel baro_channel;
 		IMUChannel x_accel, y_accel, z_accel, x_gyro, y_gyro, z_gyro;
 		Data32Channel gps_longitude, gps_latitude, gps_altitude, gps_status;
-		DigitalOutChannel out0, out1, out2, out3;
+		DigitalOutChannel out0,out1,out2,out3;
```

Comment:
Can you revert this?|
```suggestion
		DigitalOutChannel out0, out1, out2, out3;
```